docs/draft/webview.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Dapp browser API usage
+version: 0.1.0
+permalink: /spec/webview
+status: Draft
+authors: 
+---
+
+# Dapp browser API usage
+    
+##  Table of Contents
+
+1. [Abstract](#abstract)
+2. [Definitions](#definitions)
+3. [Usage](#usage)
+    * [Properties](#properties)
+        * [`isStatus`](#isStatus)
+        * [`status`](#status)
+    * [Methods](#methods)
+        * [`enable`](#enable)
+        * [`isConnected`](#isConnected)
+        * [`request`](#request)
+        * [`scanQRCode`](#scanQRCode)
+        * [`send`](#send)
+        * [`sendAsync`](#sendAsync)
+        * [`sendSync`](#sendSync)
+4. [Implementation](#implementation)
+5. [Compatibility](#compatibility)
+6. [Changelog](#changelog)
+7. [Acknowledgements](#acknowledgements)
+8. [Copyright](#copyright)
+9. [Footnotes](#footnotes)
+
+## Abstract
+
+The application should expose an Ethereum Provider object (`window.ethereum`) to JS code running inside the browser. Additionally, the browser component should also provide bidirectional communication between JS code and the application. It is important to have the `window.ethereum` object available before the page loads, otherwise Dapps might not work correctly.
```

Comment:
This isn't really an abstract, the abstract should explain what is contained in this specification.

Reply:
Right! Added an abstract and created an Overview section.